addons/web_editor/static/src/js/wysiwyg/widgets/link_popover_widget.js

```diff
@@ -61,6 +61,15 @@ const LinkPopoverWidget = Widget.extend({
         .popover('show')
         .data('bs.popover').tip.classList.add('o_edit_menu_popover');
 
+        // hide popover while typing on mega menu
+        if (this.$target.parents('.o_mega_menu').length) {
+            const showPopoverDebounced = _.debounce(() => this.$target.popover('show'), 1500);
+            this.$target.on('keydown', () => {
```

Comment:
You add the handler but you never remove it. Probably just `keydown.link_popover` is enough as you `off` all events in that `link_popover` namespace in the `destroy` method?